helm/charts/determined/templates/master-config.yaml

```diff
@@ -229,6 +229,10 @@ data:
       {{- end }}
       {{- end }}
     {{- end}}
+    {- if .Values.defaultPassword }}
```

Comment:
nice, now I think we can delete a bunch of Helm items like ```change-password.yaml``` and ```scripts/k8s-password-change```

Reply:
Do these files not have functionality outside of setting the initial password? I see "change-password" being used in a few places

Reply:
Where else do you see change-password being used? We have the CLI command ```det user change-password``` but that is unrelated to the Kubernetes options.

On Kubernetes these launch a job that runs that Python script that tries to call the Determined master API to change the password. It shouldn't be used outside of this.

Reply:
Ah I see I thought the CLI command was related. Thanks